src/components/expansion-panel/themes/light/expansion-panel.bootstrap.scss

```diff
@@ -0,0 +1,10 @@
+@use '../../../../styles/utilities/index' as *;
+
+[part='title'],
+[part~='indicator'] {
+    color: color(gray, 800);
+}
+
+[part='subtitle'] {
+    color: color(gray, 600);
+}
```

Comment:
The description text in the Bootstrap theme should use 14px Body 2, not Subtitle 2.

Reply:
And the content (body) text should use 16px Body 1, not 14px Body 2.